### ISSUE TITLE
Creating table with LIKE clauses raises an error

### ISSUE THREAD

**Issue by qoelet** [sbdchd/squawk#270]
The following SQL example

```sql
CREATE TABLE foo (id int);
CREATE TABLE bar (LIKE foo);
```

results in the following error

```Problem linting SQL files: Squawk schema failed to parse Postgres response. This indicates a bug with Squawk. Please report this error to https://github.com/sbdchd/squawk. Schema error: unknown variant `TableLikeClause`, expected `ColumnDef` or `Constraint` at line 1 column 481```
